regression/apps/Makefile

```diff
@@ -10,7 +10,7 @@ REGRESSION_BUILD_DIR ?= $(INITRAMFS)/regression
 
 TEST_APPS := signal_c pthread network hello_world hello_pie hello_c fork_c fork execve pty mongoose
 
-C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) )
+C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) ! -path "*/mongoose/*")
```

Comment:
Add a comment to make the purpose more clear.

```Makefile
# The C head and source files of all the apps, excluding the downloaded mongoose files
C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) ! -path "*/mongoose/*")
```

And are you sure that the excluded path should be `*/mongoose/*`? This means that the `http_client.c` and `http_server.c` are also excluded.

Reply:
You are right. 
`C_SOURCES := $(shell find . -type f \( -name "*.c" -or -name "*.h" \) ! -name "mongoose.c" ! -name "mongoose.h")`
should be more reasonable.